src/sentry/api/helpers/group_index/delete.py

```diff
@@ -120,6 +122,9 @@ def delete_groups(
     if not group_list:
         return Response(status=204)
 
+    if any([group.issue_category == GroupCategory.PERFORMANCE for group in group_list]):
```

Comment:
Nit: You can drop the `[]`. That allows python to stop processing as soon as it finds anything that matches the condition